### PR TITLE
Fix typo in Maxwell filter documentation

### PR DESCRIPTION
**Pull request by mscheltienne**
A couple of typos/formatting while I was looking at MNE's version of Maxwell filters.

### CODE REVIEW

**Comment by mscheltienne** [comment]
Additional points:
- `skip_by_annotation` instead of common `reject_by_annotation` in `maxwell_filter`: 

image

- Old warning in `find_bad_channels_maxwell`:

image

**Comment by drammock** [comment]
>`skip_by_annotation` instead of common `reject_by_annotation` in `maxwell_filter`

We have both `skip_` and `reject_` in various places, it seems like `skip_` is more accurate here?

**Review by drammock — APPROVED** [review]

**Review by agramfort — APPROVED** [review]

File: mne/utils/docs.py [hunk 1]
```diff
@@ -2913,8 +2913,8 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
 
 docdict['regularize_maxwell_reg'] = """
 regularize : str | None
-    Basis regularization type, must be "in" or None.
-    "in" is the same algorithm as the "-regularize in" option in
+    Basis regularization type, must be ``"in"`` or None.
+    ``"in"`` is the same algorithm as the ``"-regularize in"`` option in
```
Comment (larsoner) [inline mne/utils/docs.py:2917]:
```suggestion
    ``"in"`` is the same algorithm as the ``-regularize in`` option in
```

**Comment by larsoner** [comment]
Thanks @mscheltienne !

**Pull request merged**